### DISCUSSION TITLE
How can I get my certificate after passing the GitHub Foundations exam

### DISCUSSION THREAD

**Discussion by princemaxi** [community/community#137873] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

It's been over a week I passed the GitHub Foundations exam but till now I've not received any mail or communication to get my certificate. Also no communication from Credily too. I need help with getting my certificate

**Comment by yuriachermann** [comment]
I'm having the same problem... lmk if you receive a solution

**Reply by princemaxi** [reply]
No solution yet

**Comment by thomassiard** [comment]
Hi there!

It sounds frustrating to wait for your certification. If it’s been over a week, it’s possible there may have been a delay or a system issue. Here’s what I recommend:

Check Your Spam/Junk Folder: Sometimes, emails from Credly or GitHub can land in these folders by mistake.

Contact GitHub Support: They can provide direct assistance or escalate the issue if necessary. You can reach them at [GitHub Support](https://support.github.com/contact).

Reach Out to Credly: If GitHub confirms they sent your certification details to Credly, you could also reach out to Credly’s support team to see if they can locate the certification under your account.
